### PR TITLE
Update product versions in Secure Product Deployment

### PR DESCRIPTION
**Pull request by anjai94**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-1745.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Security Advisory WSO2-2022-1745
+category: security-announcements
+published: August 10, 2023
+version: 1.0.0
+severity: Medium
+cvss: "6.1 (CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N)"
+---
+
+# SECURITY ADVISORY WSO2-2022-1745
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2023/WSO2-2022-1745.md:10]:
```suggestion
# Security Advisory WSO2-2022-1745
```

**Review by 1uffyD9 — APPROVED** [review]
Added a small suggestion.

**Review by 1uffyD9 — APPROVED** [review]
+1 to proceed once the comment has been addressed.

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-1745.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### IMPACT
+By leveraging the XSS attack, a malicious actor can make the browser get redirected to a malicious website,
+make changes in the UI of the web page, retrieve information from the browser or harm otherwise. However,
+since all the session related sensitive cookies are set with httpOnly flag and protected, session hijacking or
+similar attacks would not be possible.
+
+### SOLUTION
+We highly recommend migrating the latest version of the WSO2 API Manager to resolve the identified vulnerability.
+
```
[the remark this answers is not in the document]
Comment (anjai94) [reply en/docs/security-announcements/security-advisories/2023/WSO2-2022-1745.md:37]: Yes this is current process. We can not provide public PR for non master fixes.

File: en/docs/security-announcements/security-advisories/2023/.gitignore
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2023/.gitignore:1]: We can add this to base `.gitignore` file.

File: en/docs/security-announcements/security-advisories/2023/2023-advisories.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ category: security-announcements
 
 # 2023 Security Advisories
 
+* [WSO2-2022-1745]({{#base_path#}}/security-announcements/security-advisories/2023/WSO2-2022-1745)
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2023/2023-advisories.md:9]: check this.

File: en/docs/security-reporting/reward-and-acknowledgement-program/hall-of-fame.md [hunk 4]
```diff
@@ -33,7 +34,7 @@ security researcher community relationship.
 | [Abdennour Chakifi](https://twitter.com/dazaii111) |
 | [Abin Joseph](https://www.facebook.com/hacker.abin1337) |
 | [Abdulrahman Nour](https://twitter.com/aboodnour) ([redforce.io](https://redforce.io/)) |
-| [Adesh Nandkishor Kolte](https://twitter.com/AdeshKolte) |
+| [Avinash Jain](https://www.linkedin.com/in/avinash-jain-54524678/) |
```
Comment (1uffyD9) [inline en/docs/security-reporting/reward-and-acknowledgement-program/hall-of-fame.md:37]: Shall we move this under ```2023 -> WSO2 Products and Infrastructure```.

File: en/docs/security-reporting/reward-and-acknowledgement-program/hall-of-fame.md [hunk 5]
```diff
@@ -90,11 +91,13 @@ security researcher community relationship.
 | [Rodrigo Duarte Favarini Silva](https://www.linkedin.com/in/rodrigofavarini/) |
 | [Simon Gerst](https://github.com/intrigus-lgtm)  |
 | [Sajibe Kanti](https://twitter.com/Sajibekantibd) |
+| [Shailendra Singh Sachan](https://www.linkedin.com/in/shailendra-singh-sachan-b8205b184)
```
Comment (1uffyD9) [inline en/docs/security-reporting/reward-and-acknowledgement-program/hall-of-fame.md:94]: Shall we move this under ```2023 -> WSO2 Products and Infrastructure```.

File: en/docs/security-reporting/reward-and-acknowledgement-program/hall-of-fame.md [hunk 6]
```diff
@@ -90,11 +91,13 @@ security researcher community relationship.
...
 | [Samir Hadji](https://twitter.com/dz_samir)  |
 | [Samitha Madhusanka](https://www.linkedin.com/in/madhusanka-athapaththu-b4b936ab/) |
 | [Sathish Kumar Balakrishnan](http://sathish.co.in/) ([Cyber Security Works Pvt Ltd](http://cybersecurityworks.com/)) |
 | [Soner Soydinc](#) |
 | [Suyog Palav](https://medium.com/@suyogpalav/) |
+| [Suraj Theekshana](https://www.linkedin.com/in/suraj-theekshana-10171023a/)
```
Comment (1uffyD9) [inline en/docs/security-reporting/reward-and-acknowledgement-program/hall-of-fame.md:100]: Shall we move this under ```2023 -> WSO2 Products and Infrastructure```.

File: en/mkdocs.yml [hunk 7]
```diff
@@ -75,6 +75,7 @@ nav:
       - '2023 Advisories':
         - '': 'security-announcements/security-advisories/2023/2023-advisories.md'
         - 'WSO2-2021-1318': 'security-announcements/security-advisories/2023/WSO2-2021-1318.md'
+        - 'WSO2-2022-1745': 'security-announcements/security-advisories/2023/WSO2-2022-1745.md'
```
Comment (1uffyD9) [inline en/mkdocs.yml:78]: This one is already in `2022` page.

**Review by 1uffyD9 — CHANGES_REQUESTED** [review]

**Review by 1uffyD9 — CHANGES_REQUESTED** [review]
have one place to check and change (if required).

File: en/docs/security-announcements/security-advisories/2022/WSO2-2021-1738.md [hunk 8]
```diff
@@ -111,17 +113,6 @@ Otherwise, you may apply the relevant fixes to the product based on the public f
 Or else you may follow the mitigation steps given below.
 
 #### GROUP 1
-For below listed product versions, it is recommended to remove all the mappings defined inside the `FileUploadConfig` tag in `<PRODUCT-HOME>/repository/conf/carbon.xml`.
```
[the remark this answers is not in the document]
Comment (1uffyD9) [reply en/docs/security-announcements/security-advisories/2022/WSO2-2021-1738.md:114]: resolved

File: en/docs/security-announcements/security-advisories/2022/WSO2-2021-1738.md [hunk 9]
```diff
@@ -111,17 +113,6 @@ Otherwise, you may apply the relevant fixes to the product based on the public f
...
-* WSO2 Identity Server 5.8.0, 5.7.0, 5.6.0, 5.5.0, 5.4.1, 5.4.0, 5.3.0, 5.2.0, and older versions
-* WSO2 Identity Server as Key Manager 5.7.0, 5.6.0, 5.5.0, 5.3.0, and older versions
-* WSO2 IS Analytics 5.6.0, 5.5.0, 5.4.1, 5.4.0, and older versions
-* WSO2 OBAM 1.5.0 and older versions
-* WSO2 OBKM 1.5.0 and older versions
-
-
-#### GROUP 2
 
 For following **WSO2 API Manager version 4.0.0, 3.2.0, 3.1.0 and 3.0.0**, it is recommand to append the following configuration lines to the `<PRODUCT-HOME>/repository/conf/deployment.toml` file and restart the server.
```
[the remark this answers is not in the document]
Comment (1uffyD9) [reply en/docs/security-announcements/security-advisories/2022/WSO2-2021-1738.md]: resolved

File: en/docs/security-announcements/security-advisories/2022/WSO2-2021-1738.md [hunk 10]
```diff
@@ -139,29 +130,10 @@ permissions = ["/permission/protected/"]
...
-http_method = "all"
- 
-[[resource.access_control]]
-context="(.*)/fileupload/(.*)"
-secure=true
-http_method = "all"
-permissions = ["/permission/protected/"]
-```
-#### GROUP 4
+#### GROUP 2
```
[the remark this answers is not in the document]
Comment (1uffyD9) [reply en/docs/security-announcements/security-advisories/2022/WSO2-2021-1738.md:136]: resolved

File: en/docs/security-announcements/security-advisories/2022/WSO2-2021-1738.md [hunk 11]
```diff
@@ -188,18 +160,18 @@ http_method = "all"
 permissions = ["/permission/protected/"]
 ```
 
-#### GROUP 5
+#### GROUP 3
```
[the remark this answers is not in the document]
Comment (1uffyD9) [reply en/docs/security-announcements/security-advisories/2022/WSO2-2021-1738.md:166]: resolved

File: en/get-pip.py
[the remark this answers is not in the document]
Comment (1uffyD9) [reply en/get-pip.py:1]: resolved

File: en/docs/security-announcements/security-advisories/2023/2023-advisories.md (hunk 3 above)
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2023/2023-advisories.md:9]: This advisory is in 2022 advisory list. please check

**Review by 1uffyD9 — CHANGES_REQUESTED** [review]

**Review by 1uffyD9 — APPROVED** [review]
Contents look good. +1

**Review by 1uffyD9 — APPROVED** [review]

**Review by ayomawdb — APPROVED** [review]

**Pull request merged**